apps/web/playwright/login.2fa.e2e.ts

```diff
@@ -162,6 +164,14 @@ test.describe("2FA Tests", async () => {
   });
 });
 
+async function removeOtpInput(page: Page) {
+  await page.locator('input[name="2fa6"]').waitFor({ state: "visible", timeout: 60_000 });
```

Comment:
This is a long timeout. Can we do 30 seconds instead?

Reply:
Done